tests/test_mutable_morphology.cpp

```diff
@@ -0,0 +1,59 @@
+#include "contrib/catch.hpp"
+
+#include <morphio/morphology.h>
+#include <morphio/mut/morphology.h>
+
+#if defined(__APPLE__)
```

Comment:
what if the *tests* (and only the tests) c++17, so that `experimental` isn't used, and std::filesystem is available?

Reply:
I would be VERY happy :D 

But does it makes sense to test the code in 17 when we release it in 11 ?

Reply:
I thought about it but it was not the best solution imao so that s why I tried to fix this with 11. (It was fine with linux but my mac is broken so this is a nightmare for the mac compilation)